sourcegraph/about-replatform
# Port community page

This closes #71. It brings in only the Community page and no other components.

- Updated links from `<a>` tags to `<Link>` components
- Made a styling change to improve spacing on the unordered list below "DM us on Slack. We're here."

## To Test
- Please head to `/community` and check that the design looks as expected
- Test the static build
- The event dates listed under "What's Next?" are outdated. Are content updates in scope for this PR?
- Does the replatform include a proposed location for tracking.ts and other future analytics-related materials?
